core/src/main/java/com/microsoft/applicationinsights/internal/config/ConfigurationFileLocator.java

```diff
@@ -86,24 +85,26 @@ public InputStream getConfigurationFile() {
         }
 
         if (configurationFile != null) {
-            InternalLogger.INSTANCE.logAlways(InternalLogger.LoggingLevel.INFO, "Configuration file has been successfully found in: '%s'", configurationFile);
+            InternalLogger.INSTANCE.info("Configuration file has been successfully found in: '%s'", configurationFile);
             try {
                 return new FileInputStream(configurationFile);
             } catch (FileNotFoundException e) {
-                InternalLogger.INSTANCE.logAlways(InternalLogger.LoggingLevel.WARN, "Configuration file '%s' could not be opened for reading", configurationFile);
-                InternalLogger.INSTANCE.trace("stack trace is %s", ExceptionUtils.getStackTrace(e));
+                InternalLogger.INSTANCE.warn("Configuration file '%s' could not be opened for reading %s", configurationFile,
+                        ExceptionUtils.getStackTrace(e));
             }
         } else {
-            InternalLogger.INSTANCE.logAlways(InternalLogger.LoggingLevel.WARN, "Configuration file '%s' could not be found", configurationFileName);
+            InternalLogger.INSTANCE.error("Configuration file '%s' could not be found", configurationFileName);
```

Comment:
Why error instead of warn?

Reply:
I thought that AI configuration file is required. I am aware that there is certain sort of default configuration but still I thought that config file has iKey. unless user passes it as an environment variable it needs to be present here.